<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Mithun-HomePC" failures="2" tests="13" name="com.test.MyTest" time="0.021" errors="0" timestamp="2021-06-29T15:24:13 IST" skipped="10">
  <testcase classname="com.test.MyTest" name="test1" time="0.000">
    <failure message="expected:&lt;false&gt; but was:&lt;true&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<false> but was:<true>
at org.testng.AssertJUnit.fail(AssertJUnit.java:65)
at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:468)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:230)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:240)
at com.test.MyTest.test1(MyTest.java:12)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1378)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test1 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test2" time="0.000">
    <failure message="expected:&lt;false&gt; but was:&lt;true&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<false> but was:<true>
at org.testng.AssertJUnit.fail(AssertJUnit.java:65)
at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:468)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:230)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:240)
at com.test.MyTest.test2(MyTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1378)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test2 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test2" time="0.001">
    <skipped/>
  </testcase> <!-- test2 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test2" time="0.001">
    <skipped/>
  </testcase> <!-- test2 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test2" time="0.002">
    <skipped/>
  </testcase> <!-- test2 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test1" time="0.001">
    <skipped/>
  </testcase> <!-- test1 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test3" time="0.001"/>
  <system-out/>
  <testcase classname="com.test.MyTest" name="test2" time="0.001">
    <skipped/>
  </testcase> <!-- test2 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test1" time="0.001">
    <skipped/>
  </testcase> <!-- test1 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test1" time="0.001">
    <skipped/>
  </testcase> <!-- test1 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test1" time="0.011">
    <skipped/>
  </testcase> <!-- test1 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test1" time="0.000">
    <skipped/>
  </testcase> <!-- test1 -->
  <system-out/>
  <testcase classname="com.test.MyTest" name="test2" time="0.001">
    <skipped/>
  </testcase> <!-- test2 -->
  <system-out/>
</testsuite> <!-- com.test.MyTest -->
